mlflow/server/js/src/i18n/default/en.json

```diff
@@ -839,6 +843,10 @@
     "defaultMessage": "Parameters",
     "description": "Table title text for parameters table in the model comparison page"
   },
+  "bUNg0K": {
+    "defaultMessage": "Perform inference via model.transform()",
+    "description": "Code comment which states on how we can perform inference"
+  },
```

Comment:
Nit:

```suggestion
  "bUNg0K": {
    "defaultMessage": "Perform inference via model.transform()",
    "description": "Code comment which states how we can perform SparkML inference"
  },
```

Reply:
This file is auto-generated. I'll modify the source.